released_plugins/v3d_plugins/MIND/soma_segmentation.pro

```diff
@@ -0,0 +1,16 @@
+
+TEMPLATE	= lib
+CONFIG	+= qt plugin warn_off
+QT += widgets
+#CONFIG	+= x86_64
+VAA3DPATH = C:/Users/shida/Offline/Projects/vaa3d/v3d_external
```

Comment:
Is there a way for this value to not be hard coded here?

Reply:
or it seems like the other plugins use directly

`VAA3DPATH =   ../../../../v3d_external`

Reply:
Ah yeah it came from VAA_3D home variable, but the generated code hard coded it. Good catch - I will fix it and merge